drivers/infiniband/hw/ntrdma/ntrdma_cmd.c

```diff
@@ -672,7 +672,7 @@ static int ntrdma_cmd_recv_none(struct ntrdma_dev *dev,
 {
 	struct ntrdma_cmd_hdr cmd;
 
-	cmd = READ_ONCE(*_cmd);
```

Comment:
1. Why not __READ_ONCE instead?
2. Why not in all cases on READ_ONCE in all the other files?